### DISCUSSION TITLE
Reflex template on replit

### DISCUSSION THREAD

**Discussion by pglevy** [reflex-dev/reflex#3647] [General]
In case anyone finds it useful, I set up a [Reflex template on replit](https://replit.com/@pglevy/Reflex-Template?v=1#README.md) so you can run a basic app and try it out with the click of a button—no local installation required.

**Comment by unoctanium** [comment]
Hi @pglevy 
I tried this template but  it seems outdated.
Did somebody get reflex running on replit?

**Reply by pglevy** [reply]
Hi @unoctanium , thanks for the mention. I had this working previously but hadn't tried it in a while. I just tried to upgrade the reflex version and run it but after trying a few things, looks like I completely borked the template. I then tried [starting from scratch](https://replit.com/@pglevy/JuicySandyProfile?v=1), using my instructions for creating the template. It seemed to go well for a while, but then I ran into the same error, which seems to have something to do with missing the `glibc` package. I tried adding the package, but this just results in different errors. So I'm not sure where to go from here, other than reaching out to Replit support.

**Reply by unoctanium** [reply]
Oh. I managed to make it work, thanks to https://github.com/reflex-dev/reflex/pull/4162/commits by @adhami3310.

Here is what I did:

- install my own node and bun versions (but no additional glibc!)
- install unzip
- install python 12.3
- set env vars `REFLEX_USE_SYSTEM_NODE` and `REFLEX_USE_SYSTEM_BUN` to 1
- installed reflex from repo: `poetry add git+https://github.com/reflex-dev/reflex.git` to get the latest changes from @adhami3310 
- init reflex with: `poetry run reflex init --loglevel debug`
- run reflex with `poetry run reflex run --loglevel debug`

Works as expected.
As soon as the latest changes are in a release, I will set up and publish a proper Replit template.
Here is my test version: https://replit.com/@unoctanium/ReflexApp?s=app

**Reply by pglevy** [reply]
Nice! Seems like it should be simpler, but at least you got it working. 🎉 Thanks for the all the detail!

**Comment by unoctanium** [comment]
With v0.63 of reflex, installation and usage became much easier and more stable under Replit.

I have published an updated template to use reflex with Replit under https://replit.com/@unoctanium/Python-Reflex?v=1
